boards/shields/nxp_m2_wifi_bt/doc/index.rst

```diff
@@ -70,3 +71,38 @@ For example:
    :board: mimxrt1060_evk@C//qspi
    :shield: nxp_m2_2el_wifi_bt
    :goals: build
+   :gen-args: -DEXTRA_CONF_FILE="nxp/overlay_hosted_mcu.conf"
+
+.. note::
+   To build Wi-Fi shell application, "nxp/overlay_hosted_mcu.conf" config file
+   must be passed along with default conf file as mentioned in above build command.
+
+Hardware Rework to Enable M.2 Interfaces
+****************************************
+
+Rework for MIMXRT1060EVK Rev-C
+==============================
```

Comment:
Would it be better to have these rework instructions in the individual board documentation page